chainerx/__init__.py

```diff
@@ -70,3 +71,9 @@ def __init__(self, *args, **kwargs):
 
 def is_available():
     return _available
+
+
+if _available and _core.is_debug():
```

Comment:
How about emitting a warning if `chainer.is_debug() != chainerx.is_debug()`?

Reply:
They are not intended to be aligned.
Perhaps it's better to change the name of `chainerx.is_debug`?

Reply:
Made it private for now as discussed, as `chainerx._is_debug`.